library/src/main/java/com/schibsted/spain/barista/assertion/BaristaHelperTextAssertions.kt

```diff
@@ -0,0 +1,42 @@
+package com.schibsted.spain.barista.assertion
+
+import android.view.View
+import androidx.annotation.IdRes
+import androidx.annotation.StringRes
+import androidx.test.InstrumentationRegistry
+import androidx.test.espresso.matcher.ViewMatchers
+import com.google.android.material.textfield.TextInputLayout
+import com.schibsted.spain.barista.internal.assertAny
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+
+object BaristaHelperTextAssertions {
+  @JvmStatic
+  fun assertHelperText(@IdRes viewId: Int, @StringRes text: Int) {
+    val resourceString = InstrumentationRegistry.getTargetContext().resources.getString(text)
+    assertHelperText(viewId, resourceString)
+  }
+
+  @JvmStatic
+  fun assertHelperText(@IdRes viewId: Int, text: String) {
+    ViewMatchers.withId(viewId).assertAny(matchError(text))
+  }
+
+  private fun matchError(expectedHelperText: String): Matcher<View> {
```

Comment:
Is there a way to differentiate between helper and error text?

Reply:
Doh! Just took the error sample and forgot to rename the method. Will change to `matchAssistiveHelperText`